Cognite.Simulator.Utils/RoutineImplementationBase.cs

```diff
@@ -176,6 +184,8 @@ private void ParseSet(Dictionary<string, string> arguments)
             var matchingInputs = _config.Inputs.Where(i => i.ReferenceId == argRefId).ToList();
             if (matchingInputs.Any() && _inputData.ContainsKey(argRefId))
             {
+                string flattenedArguments = SimulatorLoggingUtils.FlattenDictionary(extraArgs);
+                _logger.LogInformation("Setting input for Reference Id: {Input}. Arguments: {Arguments}", matchingInputs.First().ReferenceId, flattenedArguments);
```

Comment:
```suggestion
                _logger.LogDebug("Setting input for Reference Id: {Input}. Arguments: {Arguments}", matchingInputs.First().ReferenceId, flattenedArguments);
```
I think we should treat those as debug ones